sharepoint/sharepoint-ps/sharepoint-online/Copy-SPOPersonalSitePage.md

```diff
@@ -0,0 +1,186 @@
+- Move: This method creates a new copy at the new location and deletes the original page from the source.
+
+**Where can I move the existing page(s) to and from?**
+
+| Source | Destination |
+| :------------------- | :---------- | 
+| SharePoint pages library in OneDrive  | SharePoint pages library in OneDrive | 
+| SharePoint pages library in OneDrive | SharePoint site | 
+| SharePoint site    | SharePoint site | 
+
+**How do I query the code status?**
+
+After the copy is finished, you'll see the status of the code. If the copy was successful, we'll provide the URL for the new page
+
+If the copy is still in progress, you'll get a message with a URL. You can paste this link into your browser to check the status of the copy job. The JobState property will show a number indicating the job's status.
+
+```powershell
+https://contoso.microsoft.com/sites/testsite/_api/sitepages/pages/CopyToStatus('243925c7-cea7-4430-bb90-299ed9122d0b').
+```
+
+## SYNTAX
+
+### CopySinglePage
+
+```powershell
+Copy-SPOPersonalSitePage -SourceSite <SpoSitePipeBind> -DestinationSite <SpoSitePipeBind> -PageName <String> -DeleteSourcePage <Boolean> [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The Copy-SPOPersonalSitePage cmdlet allows a SharePoint Online administrator to copy one or more SharePoint pages from a selected source to a chosen destination SharePoint site. If the -DeleteSourcePage parameter is set to $true, the source page(s) will be deleted after the copy operation, effectively moving the page(s).
+
+> [!NOTE]
+> This cmdlet may not be available in all tenants as the feature rollout could be in progress. If the feature has not been enabled in your tenant, attempting to run this cmdlet will result in an error.
+
+## EXAMPLES
+
+### -----------------------EXAMPLE 1-----------------------------
+
+```powershell
+Copy-SPOPersonalSitePage -SourceSite 'https://prepspo-my.spgrid.com/personal/testuser_onmicrosoft_com' -DestinationSite 'https://prepspo.spgrid.com/sites/testsite' -PageName 'TestPage.aspx' -DeleteSourcePage $false -Confirm 
+```
+
+Example 1 demonstrates how a SharePoint Online administrator can copy the SharePoint page named `TestPage.aspx` from `testuser`'s SharePoint pages library in OneDrive to the `testsite` SharePoint site with confirmation. The source page will not be deleted.
+
+### -----------------------EXAMPLE 2-----------------------------
+
+```powershell
+Copy-SPOPersonalSitePage -SourceSite 'https://prepspo-my.spgrid.com/personal/testuser_onmicrosoft_com' -DestinationSite 'https://prepspo.spgrid.com/sites/testsite' -PageName 'TestPage.aspx' -DeleteSourcePage $true
+```
+
+Example 2 demonstrates how a SharePoint Online administrator can move the SharePoint page named `TestPage.aspx` from `testuser`'s SharePoint pages library in OneDrive to the `testsite` SharePoint site. The source page will be deleted after the copy operation.
+
+### -----------------------EXAMPLE 3-----------------------------
```

Comment:
>EXAMPLE [](http://example.com/codeflow?start=27&length=7)

add another example with `-AllPages` switch?

Reply:
We'll remove the 'copy all' support for now as that eng solution might have a long bug tail.